From 9264fe0100395ac21801e7db94ff08f60ce05502 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Fri, 18 Oct 2024 15:13:23 +0800
Subject: [PATCH 1008/1015] ARM: dts: aspeed: catalina: enable ssif alert pin

Set GPIOC7 as SSIF alert pin for improving SSIF message performance.

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index c7c9a89b9748..a4c68d21ac18 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -870,6 +870,7 @@ &i2c11 {
 	ssif-bmc@10 {
 		compatible = "ssif-bmc";
 		reg = <0x10>;
+		alert-gpios = <&gpio0 ASPEED_GPIO(C, 7) GPIO_ACTIVE_LOW>;
 	};
 };
 
-- 
2.47.3

